structures/Channel.ts

```diff
@@ -0,0 +1,56 @@
+import Base from './Base';
+//import User from './User'         doesn't exist yet
+
+// https://discord.com/developers/docs/resources/channel
+
+export enum ChannelType {
+  GUILD_TEXT,
+  DM,
+  GUILD_VOICE,
+  GROUP_DM,
+  GUILD_CATEGORY,
+  GUILD_NEWS,
+  GUILD_STORE,
+  GUILD_STAGE_VOICE = 13,
+}
+
+export interface Overwrite extends Base {
+  allow: string;
+  deny: string;
+  type: 0 | 1;
+}
+
+export interface VoiceRegion extends Base {
+  custom: boolean;
+  deprecated: boolean;
+  name: string;
+  optimal: boolean;
+  vip: boolean;
+}
+
```

Comment:
These I'd prefer to go in its own relevant PR and linked back as required

Reply:
agreed for VoiceRegion and Overwrite

Reply:
You want me to put them in another file?

Reply:
No, type them `unknown` and mark `// TODO` on the relevant line for now, we can add them back in later

Reply:
Alright that's done